README.md

```diff
@@ -98,40 +121,71 @@ After configuration, build with `cmake --build <build_dir> -- -j<nproc>`
 ### Logger tests
 
 Tests API implementation of logger verbosity and functionality.
-o <build_dir>/bin/logger_test
+
+```bash
+  <build_dir>/bin/logger_test
+```
 
 ## Running Contraction Tests
 
 ### Bilinear contraction tests
 
 Tests the API implementation of bilinear contraction algorithm with validation.
-o <build_dir>/bin/bilinear_contraction_f32_test
-o <build_dir>/bin/bilinear_contraction_f64_test
+
+```bash
+  <build_dir>/bin/bilinear_contraction_test
+```
 
 ### Scale contraction tests
 
 Tests the API implementation of scale contraction algorithm with validation.
-o <build_dir>/bin/scale_contraction_f32_test
-o <build_dir>/bin/scale_contraction_f64_test
 
-### Samples
+```bash
+  <build_dir>/bin/scale_contraction_test
+```
+
+## Running Permutation Tests
+
+### Permutation tests
+
+Tests API implementation of permutation algorithm with validation.
+
+```bash
+  <build_dir>/bin/permutation_test
+```
+
+## Samples
 
 These are stand-alone use-cases of the hipTensor contraction operations.
 
-## F32 Bilinear contraction
+### F32 Bilinear contraction
 
 Demonstrates the API implementation of bilinear contraction operation without validation.
-o <build_dir>/bin/simple_contraction_bilinear_f32
 
-## F32 Scale contraction
+```bash
+  <build_dir>/bin/simple_bilinear_contraction_<typeA>_<typeB>_<typeC>_<typeD>_compute_<computeType>
+```
+
+### F32 Scale contraction
 
 Demonstrates the API implementation of scale contraction operation without validation.
-o <build_dir>/bin/simple_contraction_scale_f32
+
+```bash
+  <build_dir>/bin/simple_scale_contraction_<typeA>_<typeB>_<typeD>_compute_<typeCompute>
+```
+
+### Permutation
+
+Demonstrates the API implementation of permutation operation without validation.
+
+```bash
+  <build_dir>/bin/simple_permutation
+```
 
 ### Build Samples as external client
 
-Client application links to hipTensor library,
-and therefore hipTensor library needs to be installed before building client applications.
+Client application links to hipTensor library, and therefore hipTensor library
```

Comment:
How about: "The client application links to hipTensor library; therefore, you must install the hipTensor library before building client applications."